projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -4997,6 +4997,17 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
         return 0;
     }
 
+    /**
+     * Returns a complete list of the elements in the current vertical dataview.
```

Comment:
Returns the current transformed paged/filtered/sorted/grouped data that is displayed in the grid.

Reply:
Requested changes resolved